wcfsetup/install/files/lib/data/comment/response/StructuredCommentResponseList.class.php

```diff
@@ -67,7 +67,6 @@ public function __construct(ICommentManager $commentManager, Comment $comment)
         $this->commentManager = $commentManager;
 
         $this->getConditionBuilder()->add("comment_response.commentID = ?", [$this->comment->commentID]);
-        $this->sqlLimit = $this->commentManager->getCommentsPerPage();
```

Comment:
Just to confirm: Is this an intentional change?

Reply:
Yes, that is intentional.